### PR TITLE
97 daily backups

### PR DESCRIPTION
**Pull request by brooklynbagel**
I think using the default bucket is okay since it's clear where the backups are going. I'll implement a CF to prune out old backups once we hit the 30 days mark

### CODE REVIEW

**Comment by brooklynbagel** [comment]
Will implement pruning old backups in a later PR

File: functions/index.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
-// exports.helloWorld = functions.https.onRequest((request, response) => {
-//  response.send("Hello from Firebase!");
-// });
+const client = new firestore.v1.FirestoreAdminClient();
+
+const now = new Date();
+const bucket = `gs://leveler-8fcc7.appspot.com/firestore_backups/${now.toISOString()}`;
+
+exports.scheduledFirestoreExport = functions.pubsub.schedule('every 24 hours').onRun((_ctx) => {
+  const projectId = process.env.GCP_PROJECT || process.env.GCLOUD_PROJECT;
```
Comment (samyoungnyc) [inline functions/index.js:10]: what is the difference between `GCP_PROJECT` AND `GCLOUD_PROJECT` ? Where do these Var's come from?

File: functions/index.js [hunk 2]
```diff
@@ -1,8 +1,27 @@
 const functions = require('firebase-functions');
+const firestore = require('@google-cloud/firestore');
 
-// // Create and Deploy Your First Cloud Functions
-// // https://firebase.google.com/docs/functions/write-firebase-functions
-//
```
Comment (samyoungnyc) [inline functions/index.js]: You can remove this cruft

File: functions/index.js [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
-// exports.helloWorld = functions.https.onRequest((request, response) => {
-//  response.send("Hello from Firebase!");
-// });
+const client = new firestore.v1.FirestoreAdminClient();
+
+const now = new Date();
+const bucket = `gs://leveler-8fcc7.appspot.com/firestore_backups/${now.toISOString()}`;
+
+exports.scheduledFirestoreExport = functions.pubsub.schedule('every 24 hours').onRun((_ctx) => {
```
Comment (samyoungnyc) [inline functions/index.js:9]: PLease write this as a function that clearly states what the function does

File: functions/index.js (hunk 2 above)
Comment (brooklynbagel) [reply functions/index.js]: weird, this says outdated, might be a previous commit. I like to rebase a bit before merging iin

Comment (brooklynbagel) [reply functions/index.js]: @samyoungnyc do you want me to make this one commit? Right now I've got one commit for the functions skeleton and anther for the backups functionality

**Pull request merged**